Update instructions how to build for Wasm (#8006)
dotnet/corert: Documentation/how-to-build-WebAssembly.md

2. To compile that to WebAssembly, run ```emcc HelloWasm.bc  -s ALLOW_MEMORY_GROWTH=1  C:\corert\bin\WebAssembly.wasm.Debug\sdk\libPortableRuntime.bc C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.bc -s WASM=1 -o HelloWasm.html``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). That will generate a .wasm file with your code as well as html and js files to run it.

# How to run a WebAssembly application #
1. Ensure you have Edge (Windows only), Chrome or [Firefox](https://www.getfirefox.com).
2. Launch HTTP server which will serve folder with generated html/wasm files. For example `http-server` from npm will serve, but any other option can do too. Without HTTP server, browser will prevent donwloading wasm file due to CORS restrictions.

>  *Note*: If you have FireFox, you can disable CORS for local files by go to `about:config`, set `privacy_file_unique_origin` to `false` and restart FireFox. Once that changes applied you will be able to open HTML files locally.

Reviewer: ```suggestion
 *Note*: If you have FireFox, you can disable CORS for local files by going to `about:config`, setting `privacy_file_unique_origin` to `false` and restarting FireFox. Once the changes are applied you will be able to open HTML files locally.
```

>  *Note*: If you have FireFox, you can disable CORS for local files by go to `about:config`, set `privacy_file_unique_origin` to `false` and restart FireFox. Once that changes applied you will be able to open HTML files locally.

Reviewer: ```suggestion
 *Note*: If you have FireFox, you can disable CORS for local files by going to `about:config`, setting `privacy_file_unique_origin` to `false` and restarting FireFox. Once the changes are applied you will be able to open HTML files locally.
```

>  *Note*: If you have FireFox, you can disable CORS for local files by going to `about:config`, setting `privacy_file_unique_origin` to `false` and restarting FireFox. Once the changes are applied you will be able to open HTML files locally.

Reviewer 2: I would change `files locally.` to `files directly from the file system without an HTTP server.`

Thanks for fixing the docs.

2. To compile that to WebAssembly, run ```emcc HelloWasm.bc  -s ALLOW_MEMORY_GROWTH=1  C:\corert\bin\WebAssembly.wasm.Debug\sdk\libPortableRuntime.bc C:\corert\bin\WebAssembly.wasm.Debug\sdk\libbootstrappercpp.bc -s WASM=1 -o HelloWasm.html``` (if emcc isn't on your path, you'll need to launch an Emscripten command prompt to do this). That will generate a .wasm file with your code as well as html and js files to run it.

# How to run a WebAssembly application #
1. Ensure you have Edge (Windows only), Chrome or [Firefox](https://www.getfirefox.com).

> 2. Launch HTTP server which will serve folder with generated html/wasm files. For example `http-server` from npm will serve, but any other option can do too. Without HTTP server, browser will prevent donwloading wasm file.

Reviewer 2: This is good, but its an option.  You dont have to use an HTTP server,  FIrefox can launch the html/wasm without one, although you'll need to change the CORS setting for local files:

![image](https://user-images.githubusercontent.com/2720041/75198146-e4ae6280-572d-11ea-8039-4e4ef9b3f951.png)

and 
https://support.mozilla.org/en-US/questions/1264280
Maybe you can add that info?

Author: @yowl thanks for suggestion, it is much more convenient when playing with WASM builds. I update docs
